gsa/src/web/graphql/tasks.js

```diff
@@ -538,3 +544,51 @@ export const useExportTasksByIds = options => {
 
   return exportTasksByIds;
 };
+
```

Comment:
I kinda like this, it's pretty straight forward. Not sure if there is a better way to handle all those nested promises (and there are going to be even more for the advanced wizard, I guess). We might need some error handling in case one of the mutations doesn't work. Not sure how to do this, though.   

And I wonder if we should put this in a separate wizard file. It is more than just task mutations after all and I think the advanced wizard will need even more different entities.

Reply:
> And I wonder if we should put this in a separate wizard file. It is more than just task mutations after all and I think the advanced wizard will need even more different entities.

That's a good idea. I was noting how long the task file was getting.

Reply:
The errors are handled already (will appear in red banner in task wizard dialog), but I wrote some tests to confirm that the entire chain will not go through if there are errors in prior steps of the chain. I didn't test every link in the chain (as they are quite similar) but it would not be difficult to add those test